### ISSUE TITLE
interpreter crash

### ISSUE THREAD

**Issue by opqrs** [WasmCert/WasmCert-Coq#5]
The non-itree interpreter in  220eb62 crashes
```bash
./wasm_interpreter tests/const.wasm hello 5
 OK
 OK
 OK
 crash
```
This is a regression.

**Comment by opqrs** [comment]
The trace looks like
```
parsing OK
instantiation OK
interpreting OK
step 0:

invoke
0with values (empty)

step 1:
normal
  local 1
  with values (empty)
    block i32
        i32.const 42
    end
  end local
with values (empty)
and store unchanged
step 2:
normal
  local 1
  with values (empty)
    label 1
    label_cont
      i32.const 42
    end label
  end local
with values (empty)
and store unchanged
step 3:
crash
with values (empty)
and store unchanged
crash
```

**Comment by raoxiaojia** [comment]
Fixed by ba11eea.
